app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java

```diff
@@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
         makeLoveButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
+                final MakeLoveDialogFragment makeLoveDialogFragment;
+                if (user.equals(currentUser)) {
+                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
```

Comment:
An email address is required to create a User object, but the dialog only
uses the username part of the object.  So we put an empty string in the To:
field.  This is the same thing we do in LovesListActivity.

On Mon, Dec 7, 2015 at 10:54 PM Devin Breen <notifications@github.com>
wrote:

> In
> app/src/main/java/org/ometa/lovemonster/ui/activities/UserLoveActivity.java
> <https://github.com/lovemonster/love-monster-android/pull/33#discussion_r46915039>
> :
>
> > @@ -75,7 +75,12 @@ private void setupMakeLoveButton() {
> >          makeLoveButton.setOnClickListener(new View.OnClickListener() {
> >              @Override
> >              public void onClick(View v) {
> > -                final MakeLoveDialogFragment makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(user);
> > +                final MakeLoveDialogFragment makeLoveDialogFragment;
> > +                if (user.equals(currentUser)) {
> > +                    makeLoveDialogFragment = MakeLoveDialogFragment.newInstance(new User("foo@example.com", ""));
>
> Is the foo part just a placeholder that gets overwritten?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lovemonster/love-monster-android/pull/33/files#r46915039>
> .
>